app/design/frontend/base/default/template/sailthru/horizon.phtml

```diff
@@ -50,6 +50,7 @@
 <meta name="sailthru.image.full" content="<?php echo htmlspecialchars($product->getImageUrl()); ?>" />
 <meta name="sailthru.image.thumb" content="<?php echo htmlspecialchars($product->getThumbnailUrl(50, 50)); ?>" />
 <meta name="sailthru.price" content="<?php echo htmlspecialchars($product->getPrice()*100); ?>" />
+<meta name="sailthru.special_price" content="<?php echo htmlspecialchars($product->getSpecialPrice()); ?>" />
```

Comment:
We don't need this here. 

This form adds meta tags for the 'spider'. That spider is only run once, after the initial pageview. As special price is by definition temporary, it would only be passed in via the purchase api.